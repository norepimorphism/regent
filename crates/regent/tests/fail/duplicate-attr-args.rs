fn main() {
    // Duplicate path, duplicate value (2).
    #[regent::bitwise(size = 1, size = 1)]
    struct A;
    // Duplicate path, unique values (2).
    #[regent::bitwise(size = 1, size = 2)]
    struct B;
    // Duplicate path, duplicate value (3).
    #[regent::bitwise(size = 1, size = 1, size = 1)]
    struct C;
    // Duplicate path, unique values (3).
    #[regent::bitwise(size = 1, size = 2, size = 3)]
    struct D;
}
