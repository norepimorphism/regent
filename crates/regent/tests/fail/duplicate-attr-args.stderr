error: argument is a duplicate
 --> tests/fail/duplicate-attr-args.rs:3:33
  |
3 |     #[regent::bitwise(size = 1, size = 1)]
  |                                 ^^^^

error: argument is a duplicate
 --> tests/fail/duplicate-attr-args.rs:6:33
  |
6 |     #[regent::bitwise(size = 1, size = 2)]
  |                                 ^^^^

error: argument is a duplicate
 --> tests/fail/duplicate-attr-args.rs:9:33
  |
9 |     #[regent::bitwise(size = 1, size = 1, size = 1)]
  |                                 ^^^^

error: argument is a duplicate
  --> tests/fail/duplicate-attr-args.rs:12:33
   |
12 |     #[regent::bitwise(size = 1, size = 2, size = 3)]
   |                                 ^^^^
